web-client/src/components/form/Login.tsx

```diff
@@ -37,8 +39,13 @@ const Login: React.FC = () => {
     },
     validationSchema: loginSchema,
     onSubmit: async (values: LoginFormValues) => {
+      // Update token
+      setAuthToken(authToken);
```

Comment:
I think we shouldn't set a new token from the front, as it would not be httpOnly anymore